### PR TITLE
Fixed button not visible completely #633

### LINKED ISSUE #633: Button not visible completely
**Description**
The button on https://meshery.io/calendar gets cut halfway.

**Expected Behavior**
Make the button visible completely.

**Screenshots**
Button

**Enviroment:**
 - OS: [e.g. Ubuntu]
 - Browser: [e.g. Chrome, Safari]
 - Version: [e.g. 22]
 - Device: [e.g. laptop, iPhone 8]

---
[Optional] **To Reproduce**

[Optional] **Additional Context**

### Contributor Resources

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- See Meshery site designs in this [Figma designfile](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=110%3A1). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Ramreiso**
Signed-off-by: Ramreiso Kashung ramreisog24@gmail.com

**Description**

This PR fixes #633

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ #633 ] Yes, I signed my commits.
 

] Yes, I signed my commits.
 

] Yes, I signed my commits.
 

Screenshot from 2021-12-14 10-49-36

### CODE REVIEW

**Review by warunicorn19 — CHANGES_REQUESTED** [review]
Could you please get the text to be vertically centred in the button. Also, while you're in there, I just noticed that the spelling of "calendar" is wrong in the dropdown, could you please fix that too? 
Screenshot 2021-12-14 at 5 44 19 AM

**Review by YashKamboj — CHANGES_REQUESTED** [review]
@Ramreiso  Instead of changing the padding , why don't you justify the height of the input , and use "rem"

**Comment by Ramreiso** [comment]
@warunicorn19 @YashKamboj 

I have updated the PR in response to changes requested.
Additionally, I have also fixed another bug where 'Participate' button was not visible completely at https://meshery.io/programs/gsoc/2020 .

Further requests or suggestions are most welcome!

**Comment by YashKamboj** [comment]
I think this is ready to merge

**Comment by YashKamboj** [comment]
I think this is ready to merge

**Comment by YashKamboj** [comment]
I think this is ready to merge

**Comment by YashKamboj** [comment]
I think this is ready to merge

File: calendar.html [hunk 1]
```diff
@@ -19,7 +19,7 @@ <h2>Engage in the Community</h2>
   </div>
   <a href="http://slack.layer5.io/" style="text-decoration: none !important;">
     <h3 style="text-align:center;color:aliceblue;font-size: 1.6rem;">
-      <input style="padding-bottom:.9rem;padding-top:.9rem; margin:auto; width:80%;" type="submit" value="Join the Community">
+      <input style="padding-bottom:3rem; padding-top:0.9rem; margin:auto; width:80%;" type="submit" value="Join the Community">
     </h3>
```
Comment (shubham-tyagi) [inline calendar.html:23]: Everything looks good, but maybe if we can avoid inline styling that would be great.

**Review by shubham-tyagi — CHANGES_REQUESTED** [review]

File: calendar.html (hunk 1 above)
Comment (shubham-tyagi) [reply calendar.html:23]: what do you think about this @warunicorn19?

Comment (YashKamboj) [reply calendar.html:23]: @shubham-tyagi Most files have inline styling .

Comment (warunicorn19) [reply calendar.html:23]: Yeah @YashKamboj is right, generally we don't want to do inline styling, but here, since it's already there we'll let it be

**Pull request merged**